views/includes/documents/Tampermonkey-for-Edge.md

```diff
@@ -0,0 +1,80 @@
+
+Sometimes, when you use more than one userscript on the same web page, they need to run in a particular order. You can change the order using the Tampermonkey dashboard. In the "Sort order" column, click on the triple-line icon for the script you want to move, move the mouse up or down to change the order, then click again.
+
+### More
+
+* [Get Tampermonkey from the Windows 10 Store website][edgeAddons]
+* [Tampermonkey for Chrome][tampermonkeyForChrome]
+* [Tampermonkey for Android][tampermonkeyForAndroid]
+* [Tampermonkey.net][tampermonkeyNet] - documentation, discussion and downloads for other versions of Tampermonkey.
+
+[githubFavicon]: https://assets-cdn.github.com/favicon.ico
+[oujsFavicon]: https://raw.githubusercontent.com/OpenUserJs/OpenUserJS.org/master/public/images/favicon16.png
+[oujs]: https://openuserjs.org/
+[tampermonkeyIcon]: https://raw.githubusercontent.com/wiki/OpenUserJS/OpenUserJS.org/images/tampermonkey_icon.png "Tampermonkey"
+[chrome]: Chrome
+[tampermonkeyForChrome]: Tampermonkey-for-Chrome
+[tampermonkeyForAndroid]: Tampermonkey-for-Android
+[edgeAddons]: https://www.microsoft.com/store/apps/9NBLGGH5162S
+[violentMonkeyForOpera]: Violentmonkey-for-Opera
+[edgeAddonsScreenshot1]: https://raw.githubusercontent.com/wiki/OpenUserJS/OpenUserJS.org/images/tampermonkey_edge1.png "Tampermonkey in the Windows Store website"
+[tampermonkeyEdgeScreenshot1]: https://raw.githubusercontent.com/wiki/OpenUserJS/OpenUserJS.org/images/tampermonkey_edge2.png "Tampermonkey ready to launch"
+[tampermonkeyEdgeScreenshot2]: https://raw.githubusercontent.com/wiki/OpenUserJS/OpenUserJS.org/images/tampermonkey_edge3.png "Edge Extension prompt for Tampermonkey"
+[tampermonkeyEdgeScreenshot3]: https://raw.githubusercontent.com/wiki/OpenUserJS/OpenUserJS.org/images/tampermonkey_edge4.png "Tampermonkey installed in Edge"
+[tampermonkeyEdgeScreenshot4]: https://raw.githubusercontent.com/wiki/OpenUserJS/OpenUserJS.org/images/tampermonkey_edge5.png "Tampermonkey options in Edge"
+[oujsScriptPageScreenshot1]: https://raw.githubusercontent.com/wiki/OpenUserJS/OpenUserJS.org/images/openuserjs_script.gif "Ready to install a script"
+[tampermonkeyEdgeScreenshot4]: https://raw.githubusercontent.com/wiki/OpenUserJS/OpenUserJS.org/images/tampermonkey_edge7.png "Installing a script in Edge"
+[tampermonkeyEdgeScreenshot5]: https://raw.githubusercontent.com/wiki/OpenUserJS/OpenUserJS.org/images/tampermonkey_edge5.png "Tampermonkey Dashboard"
+[tampermonkeyForAndroid]: Tampermonkey-for-Android
+[tampermonkeyForChrome]: Tampermonkey-for-Chrome
+[tampermonkeyNet]: http://tampermonkey.net/
```

Comment:
I'll go ahead and get that... need to remove a few things across the docs. Thanks for your cool addition.